docs/services/elections_bulletin_board.md

```diff
@@ -0,0 +1,36 @@
+# Elections Bulletin Board
+
```

Comment:
Please add a line like: "This is a work in progress and is not fully working yet."